### PR TITLE
Install Oracle JDK 6, 7, and 8 on vsts standard image

### PR DESCRIPTION
**Pull request by yacaovsnc**
1. Install Oracle JDK 6, 7, and 8.
2. Setup environment variables used by VSTS Java (Ant/Maven/Gradle) tasks.

### CODE REVIEW

File: ubuntu/derived/standard/Dockerfile.template [hunk 1]
```diff
@@ -23,14 +23,28 @@ RUN echo "deb [arch=amd64] http://apt-mo.trafficmanager.net/repos/dotnet-release
...
-    # Java and related build tools
-    openjdk-$(OPENJDK_VERSION)-jdk \
-    ant \
-    ant-optional \
     # .NET Core SDK
     dotnet-dev-1.0.1 \
  && rm -rf /var/lib/apt/lists/*
 
+# Install Oracle JDKs
+RUN echo debconf shared/accepted-oracle-license-v1-1 select true | debconf-set-selections
```
Comment (stepro) [inline ubuntu/derived/standard/Dockerfile.template]: Is there a reason this first line is in its own RUN statement? I'd prefer to not add a separate image layer just for this.

File: ubuntu/derived/standard/Dockerfile.template [hunk 2]
```diff
@@ -55,5 +69,9 @@ ENV ANT_HOME=/usr/share/ant \
     dotnet=/usr/bin/dotnet \
     GRADLE_HOME=/usr/share/gradle \
     grunt=/usr/local/bin/grunt \
-    JAVA_HOME=/usr/lib/jvm/java-$(OPENJDK_VERSION)-openjdk-amd64 \
     M2_HOME=/usr/share/maven \
+    JAVA_HOME_6_X64=/usr/lib/jvm/java-6-oracle \
```
Comment (stepro) [inline ubuntu/derived/standard/Dockerfile.template]: Sorry to be picky, but can we keep this list of environment variables in alphabetical order? Thanks!

File: ubuntu/derived/standard/README.md [hunk 3]
```diff
@@ -4,7 +4,7 @@ The Ubuntu-based standard images include the following tools:
 - Basic command-line utilities (curl, ftp, etc.)
 - Essential build tools (gcc, make, etc.)
 - Python and Python 3
-- OpenJDK 7 (Ubuntu 14.04) or 8 (Ubuntu 16.04)
+- Oracle JDK 6, 7 (Default JDK on 14.04) and 8 (Default JDK on 16.04)
```
Comment (stepro) [inline ubuntu/derived/standard/README.md]: This information is also in the main repo's README.md file - can you update it there as well?

Comment (yacaovsnc) [inline ubuntu/derived/standard/README.md]: Done.  Thanks!

**Comment by stepro** [comment]
I'm finalizing the update and building and pushing images, you can expect the Docker Hub images to be updated in about two hours.

**Comment by stepro** [comment]
I needed to refactor how we build the images because the standard images now take significantly longer to build. ETA is now later today.

**Comment by stepro** [comment]
All the images have been updated on Docker Hub.

**Pull request merged**